backend/src/main/java/com/redhat/cloud/notifications/routers/internal/InternalResource.java

```diff
@@ -402,19 +405,22 @@ public boolean deleteDefaultBehaviorGroup(@PathParam("id") UUID id) {
     @RolesAllowed(ConsoleIdentityProvider.RBAC_INTERNAL_ADMIN)
     public Response updateDefaultBehaviorGroupActions(@PathParam("behaviorGroupId") UUID behaviorGroupId, List<RequestDefaultBehaviorGroupPropertyList> propertiesList) {
         if (propertiesList == null) {
-            throw new BadRequestException("The request body must contain a list of EmailSubscriptionProperties");
+            throw new BadRequestException("The request body must contain a list of RequestDefaultBehaviorGroupPropertyList");
         }
 
         if (propertiesList.size() != propertiesList.stream().distinct().count()) {
-            throw new BadRequestException("The list of EmailSubscriptionProperties should not contain duplicates");
+            throw new BadRequestException("The list of RequestDefaultBehaviorGroupPropertyList should not contain duplicates");
         }
 
-        List<Endpoint> endpoints = propertiesList.stream().map(p -> {
-            EmailSubscriptionProperties properties = new EmailSubscriptionProperties();
+        List<Endpoint> endpoints = new ArrayList<>();
+        for (RequestDefaultBehaviorGroupPropertyList p : propertiesList) {
+            SystemSubscriptionProperties properties = new SystemSubscriptionProperties();
             properties.setOnlyAdmins(p.isOnlyAdmins());
             properties.setIgnorePreferences(p.isIgnorePreferences());
-            return endpointRepository.getOrCreateEmailSubscriptionEndpoint(null, null, properties);
-        }).collect(Collectors.toList());
+            endpoints.add(endpointRepository.getOrCreateSystemSubscriptionEndpoint(null, null, properties, EMAIL_SUBSCRIPTION));
+            endpoints.add(endpointRepository.getOrCreateSystemSubscriptionEndpoint(null, null, properties, DRAWER));
+        }
```

Comment:
I'm pretty sure someone will want a default behavior group that has different settings between emails and the drawer. I don't think we can use the same settings for both.

Reply:
Lets leave the `DRAWER` out of this operation for now.

We can update it later to accept these properties as params.

Reply:
That was our conclusion after a quick meeting about this topic 😄